addons/vehicles/CfgVehicles.hpp

```diff
@@ -27,6 +28,7 @@ class CfgVehicles {
         class EventHandlers;
     };
     class CLASS(Offroad_I_Black): Offroad_01_unarmed_base_F {
+        MACRO_SET_VEHICLE_CARGO
```

Comment:
What does this contain that doesn't fit a PMC?

Reply:
Misremembered on my part, the offroad actually already comes with 4 First Aid Kits, so I'll remove the macro here.